packages/proxy/src/proxy.ts

```diff
@@ -175,6 +175,18 @@ export async function proxyV1({
     proxyHeaders[FORMAT_HEADER],
   );
 
+  let orgName = proxyHeaders[ORG_NAME_HEADER];
+
+  const pieces = url
+    .split("/")
+    .filter((p) => p.trim() !== "")
+    .map((d) => decodeURIComponent(d));
```

Comment:
should we lowercase everything too?

Reply:
org names are case sensitive?

Reply:
True. I was thinking about the `btorg` part, maybe we just lowercase that comparison.

Reply:
done